.codecov.yml

```diff
@@ -2,9 +2,16 @@
 # Validation check:
 # $ curl --data-binary @.codecov.yml https://codecov.io/validate
 
+
+# https://docs.codecov.io/docs/codecovyml-reference
 codecov:
+  token: "d0fcb731-42af-465d-b680-c1ccdc824bdc"
```

Comment:
I think we are supposed to keep these tokens secret (we need to re-generate them if that's the case).

Given that you are using the token as an env var below (`$CODECOV_TOKEN`). Do you even need to configure it here?

Reply:
no, it was just a duplicate and I forget to remove it... Done now :]